portfolio-project-react/src/components/Picture.jsx

```diff
@@ -0,0 +1,15 @@
+
+const Picture = ({ url, size }) => {
```

Comment:
what if i want a separate height and width of the image?

Reply:
For that I can pass seperate props for width and height, and than set image's dimensions accordingly, but requirement mentioned in acceptance criterea was to pass single property i.e. size to specify both width and height of image.